### ISSUE TITLE
[css-fonts-4] [varfont] Allow access to named instances outside of @font-face

### ISSUE THREAD

**Issue by rsheeter** [w3c/csswg-drafts#2972]
#525 added access to named instances by specifying `font-named-instance` in `@font-face`. 

For other aspects of a variable font you can specify on @font-face OR elsewhere. For example, `font-variation-settings` (https://drafts.csswg.org/css-fonts-4/#font-variation-settings-def) will let you use the font-face with different axis positions and `font-display` can be changed outside the `@font-face` with `@font-feature-values` (https://drafts.csswg.org/css-fonts-4/#font-display-font-feature-values).

Is it plausible that a mechanism be defined such that a `font-named-instance` be set outside of `@font-face`? Just allow `font-named-instance` in more places similar to `font-variant-alternates`?

For Google Fonts and other central services this would be very convenient: ask for a variable family and then access all sorts of fun parts of it. For example:

```
@import url('https://fonts.googleapis.com/css?family=Markazi+Text');

body { font-family: 'Markazi Text'; }

/* Here I can change axis positions (font-variation-settings) or set a font-display using @font-feature-settings but I can't change my axis positions using the named instances carefully defined by my type designer 😞 */
h1 { font-named-instance: 'Markazi Text Light'; }
h2 { font-named-instance: 'Markazi Text Bold'; }
```

**Comment by litherum** [comment]
Any solution for `font-named-instance` needs to work with fallback. Because of content blockers or network errors, the page author doesn't know exactly which font is being used on a particular element. `font-named-instance` was scoped to a particular `@font-face` block so it wouldn't apply across the entire font fallback list.

**Comment by svgeesus** [comment]
I agree with @litherum that this needs to be a descriptor tied to a specific `@font-face` for much the same reason that `@font-feature-values` is tied to a specific face (because stylistic sets are not portable between fonts and don't work correctly with fallback).

Also we need to retain the CSS model with independently-variable `font-weight`, `font-style`, font-stretch` etc within a family, rather than the "big list of font instances" model which does not allow for modification of an inherited font on a child element.

**Comment by rsheeter** [comment]
To give context, I'd love for all users of Markazi Text to share cache hits. If I have to modify the `@font-face` to produce a named instance then I have to either not support named instances in Google Fonts or add query params that bust the caching. Perhaps there is no realistic way to avoid this though?

**Comment by davelab6** [comment]
> not support named instances in Google Fonts 

I think this is fine; I think VF is a good opportunity to move designers away from using named styles and towards "relative typography" or "dynamic typography" with  the advantages of "cascading" style sheets.
